temporal-sdk/src/main/java/io/temporal/internal/replay/ReplayWorkflowContext.java

```diff
@@ -260,6 +252,14 @@ void getVersion(
   /** @return replay safe UUID */
   UUID randomUUID();
 
+  /**
+   * Used to retrieve search attributes.
+   *
+   * @return SearchAttribute object which can be used by {@code
```

Comment:
Why it's `code` and not a `link`? Also, the class `WorkflowUtils` doesn't exist. This comment doesn't make much sense to me.